Detectors/MUON/MID/Simulation/test/testSimulation.cxx

```diff
@@ -392,7 +392,7 @@ BOOST_DATA_TEST_CASE(MID_SingleCluster, boost::unit_test::data::make(getDEList()
     int nGenClusters = 1, nRecoClusters = 0;
     simDigitizer.digitizer.process(hits, digitStoreMC, digitLabelsMC);
     rofRecords.clear();
-    rofRecords.emplace_back(10 * ievent, EventType::Standard, 0, digitStoreMC.size());
+    rofRecords.emplace_back(25 * ievent, EventType::Standard, 0, digitStoreMC.size());
```

Comment:
I assume the 25 here means 25ns ? Might want to use a constant to make it clear what this 25 really is.

Reply:
Hi,
If it is indeed a BC duration, it is better to use ``o2::lhc::LHCOrbitNS`` from 
``Common/Constants/include/CommonConstants/LHCConstants.h``

Reply:
Hi @aphecetche and @shahor02 ,
yes, indeed this is the BC spacing: I want the data to belong to different BCs.
So I replaced it with:
`o2::constants::lhc::LHCBunchSpacingNS`
and not with LHCOrbitNS, since AFAIK, this would increase the orbit by one, not the BC.
Cheers,
Diego